cognite/powerops/custom_modules/power_model_v1/data_models/views/bid_configuration/1-interface.PartialBidConfiguration.view.yaml

```diff
@@ -0,0 +1,41 @@
+space: "{{powerops_models}}"
+externalId: PartialBidConfiguration
+name: PartialBidConfiguration
+description: A partial bid configuration
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{powerops_models}}"
+          externalId: PartialBidConfiguration
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{powerops_instance_space}}"
+version: "{{version}}"
+properties:
+  name:
+    container:
+      space: "{{powerops_models}}"
+      externalId: PartialBidConfiguration
+      type: container
+    containerPropertyIdentifier: name
+    name: name
+    description: Name for the PartialBidConfiguration
+  method:
+    container:
+      space: "{{powerops_models}}"
+      externalId: PartialBidConfiguration
+      type: container
+    containerPropertyIdentifier: method
+    name: method
+    description: Name of the method used for the bid calculation
+  addSteps:
```

Comment:
Nice with addSteps here. When creating the bid matrices, then we can transform this boolean into the following step state strings: "to be applied", "is applied", or "not applicable".